man/org.freedesktop.systemd1.xml

```diff
@@ -202,6 +202,10 @@ node /org/freedesktop/systemd1 {
       DisableUnitFiles(in  as files,
                        in  b runtime,
                        out a(sss) changes);
+      DisableUnitFilesWithInstallInfo(in  as files,
+                                      in  b runtime,
```

Comment:
model this after `DisableUnitFilesWithFlags`, and add a flags parameter instead of `in b runtime`, so that next time it can be extended using a flag (also move this down after that one)